### DISCUSSION TITLE
Is it possible to buffer and record before a motion event is triggered?

### DISCUSSION THREAD

**Discussion by jsphonorio** [roflcoopter/viseron#864] [Q&A] [answered]
Trying to figure out if this feature exists. Sometimes I have a motion go off, but I would like to see what happened right before the motion. Similar to how some video game capture software does save the last 30 seconds after pressing the hotkey. It would need some sort of buffer that's constantly recording and then deleting, but then get saved if a motion trigger happens.

**Comment by roflcoopter** [comment, marked as answer]
Yes that can be configured using the `lookback` option under `recorder` for each camera

**Comment by jsphonorio** [comment]
Ah thanks!
